packages/twenty-server/src/database/typeorm/core/migrations/common/1757364323932-1757000000000-addApplicationEntityAndRelationships.ts

```diff
@@ -0,0 +1,192 @@
+import { type MigrationInterface, type QueryRunner } from 'typeorm';
+
+export class AddApplicationEntityAndRelationships1757364323932
+  implements MigrationInterface
+{
+  name = 'AddApplicationEntityAndRelationships1757364323932';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            DO $$ BEGIN
+                CREATE TYPE "core"."application_sourcetype_enum" AS ENUM('git', 'local', 'marketplace');
+            EXCEPTION
+                WHEN duplicate_object THEN null;
+            END $$;
+        `);
+
+    await queryRunner.query(`
+            CREATE TABLE IF NOT EXISTS "core"."application" (
+                "id" uuid NOT NULL DEFAULT uuid_generate_v4(),
+                "universalIdentifier" uuid NOT NULL,
+                "label" character varying NOT NULL,
+                "description" text,
+                "icon" character varying,
```

Comment:
Let's remove icon for now? We need to handle static assets import cleanly first probably